package.json

```diff
@@ -15,7 +15,8 @@
     "fuzzaldrin-plus": "^0.1.0",
     "underscore-plus": "*",
     "element-resize-detector": "*",
-    "etch": "<0.9"
+    "etch": "<0.9",
+    "fastdom": "*"
```

Comment:
Well, the point is that we have to do loads of DOM reads and writes to update each result in an editor. These *absolutely* will need to get batched, because otherwise performance will be horrible. That said, I think I last benchmarked the need for `fastdom` before using `process.nextTick` so, I'll check again.